### DISCUSSION TITLE
fallbackClusters 对于envoy版本的要求1.20.6以上吗？

### DISCUSSION THREAD

**Discussion by aseTo2016** [alibaba/higress#429] [Q&A]
请问fallbackClusters 对于envoy版本的要求1.20.6以上吗？
现在打包脚本里都是使用的1.20版本，打出来版本之后，支持不了fallbackClusters的功能
istio-discovery里会打印proxy  support fallback false的日志
“log.Debugf("Envoy version is %s", versionStr)” 这一行没有执行，说明ENVOY_VERSION这个也没有
ENVOY_VERSION是由envoy传递过来的吗？在envoy源码里面没有看到

var notSupportFallback, _ = version.NewVersion("1.20.6")
func supportFallback(proxy *model.Proxy) (isSupport bool) {
	defer func() {
		log.Debugf("proxy %s support fallback %v", proxy.ID, isSupport)
	}()

	rawVersion, exist := proxy.Metadata.Raw["ENVOY_VERSION"]
	if !exist {
		return
	}

	versionStr := rawVersion.(string)
	log.Debugf("Envoy version is %s", versionStr)
	curVersion, err := version.NewVersion(versionStr)
	if err != nil {
		return
	}

	isSupport = curVersion.GreaterThan(notSupportFallback)
	return
}

**Comment by johnlanni** [comment]
嗯 已经有PR在修复这个问题

**Reply by johnlanni** [reply]
https://github.com/alibaba/higress/pull/575 已经修复
